src/amo/css/App.scss

```diff
@@ -27,7 +27,24 @@ body {
 }
 
 .App-content {
-  padding: 0 0 10px;
+  background: $accent-color;
+
+  > div {
+    padding: 10px;
+  }
+
+  > .full-width {
```

Comment:
Looking again though... some of the overrides I added as a precaution and we don't need them.

But it seems a cleaner solution (and one that doesn't cause odd breaks when content at the start inside `.App-content` has margins) than applying padding to every element inside `.App-content` manually.